### DISCUSSION TITLE
Welcome to co10_Escape Discussions!

### DISCUSSION THREAD

**Discussion by CaptainPStar** [NeoArmageddon/co10_Escape#104] [General]
Generic Github Discussion starter Template:

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Pinkyyyy** [comment]
Hi, im Pinky, i uhm dont do much coding or scripting, i mostly bother the poor people in arma 3 discord to help me get my mission scripts to work. Escape might be my favourite gamemode for arma 3, i love the idea of starting with nothing and having to scramble for resources and also the randomness of it all. Id love to learn more about how to help with the project maybe learn how to port maps and what not.
Im excited for the future of this project!
